backend/src/main/java/gov/cdc/usds/simplereport/api/queue/QueueMutationResolver.java

```diff
@@ -34,21 +37,25 @@ public QueueMutationResolver(TestOrderService tos, PersonService ps) {
       _ps = ps;
     }
 
-    public void addTestResult(String deviceID, String result, String patientID, String dateTested) {
+    public void addTestResult(String deviceID, String result, String patientID, String dateTested) throws ParseException {
+      DateFormat df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
+      Date dateTestedString = df.parse(dateTested);
```

Comment:
In constructive commentary, I will note that "DateTime" is a scalar type in the graphql schema so you can probably just accept a Date object as the method parameter instead.

In less constructive commentary, we have here a `String` called `dateTested` and a `Date` called `dateTestedString`.  That is... reverse hungarian notation perhaps? 🤣